frontend/.env.example

```
@@ -1,16 +1,27 @@
+# API_BASE_URL: APIサーバーのベースURL
+#
+# ■ FLAVOR=dev (開発環境) の場合:
+#   値を「空」にすると、実行OSに応じて以下のローカルURLに自動フォールバックされる
+#   - Androidエミュレータ: http://10.0.2.2:8000
+#   - iOSシミュレータ:     http://localhost:8000
+#   ※ 特定のローカルIPや外部サーバーに接続したい場合のみ、URLを記述すること
+#
+# ■ FLAVOR=prod (本番環境) の場合:
+#   必須項目。本番用のAPI URL（例: https://hoge-fuga.run.app）の指定が必要
+#
+API_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)